java/akka/smoke_test/akka_test.go

```diff
@@ -0,0 +1,126 @@
+package akka_test
+
+import (
+	"flag"
+	"fmt"
+	"github.com/paketo-buildpacks/samples/tests"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+var builders tests.BuilderFlags
+
+func init() {
+	flag.Var(&builders, "name", "the name a builder to test with")
+}
+
+func TestAkka(t *testing.T) {
+	Expect := NewWithT(t).Expect
+
+	Expect(len(builders)).NotTo(Equal(0))
+
+	SetDefaultEventuallyTimeout(60 * time.Second)
+
+	suite := spec.New("Java - Akka", spec.Parallel(), spec.Report(report.Terminal{}))
+	for _, builder := range builders {
+		suite(fmt.Sprintf("Akka with %s builder", builder), testAkkaWithBuilder(builder), spec.Sequential())
+	}
+	suite.Run(t)
+}
+
+func testAkkaWithBuilder(builder string) func(*testing.T, spec.G, spec.S) {
+	return func(t *testing.T, context spec.G, it spec.S) {
+		var (
+			Expect     = NewWithT(t).Expect
+			Eventually = NewWithT(t).Eventually
+
+			pack   occam.Pack
+			docker occam.Docker
+			home   string = os.Getenv("HOME")
+		)
+
+		it.Before(func() {
+			pack = occam.NewPack().WithVerbose().WithNoColor()
+			docker = occam.NewDocker()
+		})
+
+		context("detects a Java app", func() {
+			var (
+				image     occam.Image
+				container occam.Container
+
+				name   string
+				source string
+			)
+
+			it.Before(func() {
+				var err error
+				name, err = occam.RandomName()
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			it.After(func() {
+				err := docker.Container.Remove.Execute(container.ID)
+				if err != nil {
+					Expect(err).To(MatchError("failed to remove docker container: exit status 1: Container name cannot be empty"))
+				} else {
+					Expect(err).ToNot(HaveOccurred())
+				}
+
+				Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
+
+				err = docker.Image.Remove.Execute(image.ID)
+				if err != nil {
+					Expect(err.Error()).To(ContainSubstring("failed to remove docker image: exit status 1: Error"))
```

Comment:
```suggestion
					Expect(err).To(MatchError(ContainSubstring("failed to remove docker image: exit status 1: Error")))
```

I'm pretty sure You can nest matchers so you can get substring match on an error.